source/extensions/dynamic_modules/sdk/go/abi.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
```

Comment:
reviewers: this file is identical to source/extensions/dynamic_modules/abi.h as per the comment, so you can ignore.